tesseract-server/src/handlers/aggregate.rs

```diff
@@ -91,7 +91,7 @@ pub fn do_aggregate(
         .and_then(move |df| {
             let content_type = format_to_content_type(&format);
 
-            match format_records(&headers, df, format) {
+            match format_records(&headers, df, format, None) {
```

Comment:
@Jayrmajithia can you confirm that this should this be none?

Reply:
That should be none so, when we don't pass the Source data it will not be generated

Reply:
I was thinking we might want to have source data in the response in this case because it's a data aggregation endpoint?

Reply:
So, We have to modify the format_recode method because the data response is in format of the Dataframe. And Source will not be in the format of the Dataframe. if append in it will be added to the data property and won't create the Source property so if change the method format_recode where we pass the Source data too it will be appended in the result for the data aggregation endpoint but rest of the place where we are using the formamt_record method has to pass None

Reply:
Not sure I follow here? Would you be able to clarify?

Reply:
the result from the handler/aggregate and other modules where we use this format_records method have the result in the format of Dataframe with this method it is converted to the JSON strings where the data is the only property(i.e. the output will say { data: [result]} ) present. So, when we want the source property to be added in the output result we pass the source data but if we don't want to pass the source data we just put the value None. Hope this clarifies

Reply:
(As a summary: we did change this behavior so that sources will also be available from non-logic layer endpoints)